app/src/main/java/org/asteroidos/sync/MainActivity.java

```diff
@@ -117,26 +140,32 @@ public void onEvent(BleManager.StateListener.StateEvent event) {
     @Override
     public void onDestroy() {
         super.onDestroy();
-        if(mStatus != SynchronizationService.STATUS_CONNECTED)
+        if (mStatus != SynchronizationService.STATUS_CONNECTED)
             stopService(mSyncServiceIntent);
     }
 
     /* Fragments switching */
     @Override
-    public void onDefaultDeviceSelected(String macAddress) {
+    public void onDefaultDeviceSelected(BluetoothDevice mDevice) {
+        scanner.stopScan(scanCallback);
+        mListFragment.scanningStopped();
         mDetailFragment = new DeviceDetailFragment();
 
+        if(mListFragment != null)        mListFragment.scanningStopped();
```

Comment:
it's a bit confusing that this PR contains a lot of random cleanups like adding spaces after if here and there but also introduces new code that doesn't respect that same convention. let's pick one style and stick to it. also please in the future keep cleanups clearly separate from refactorings